compiler/src/dmd/globals.d

```diff
@@ -467,6 +467,17 @@ extern (C++) struct Global
         return major * 1000 + minor;
     }
 
+    /**
+     * Indicate to stateful error sinks that no more errors can be produced.
+     * This is to support error sinks that collect information to produce a
+     * single (say) report.
+     */
+    extern(C++) void plugErrorSinks()
```

Comment:
update headers, `frontend.h` and `globals.h`

Reply:
done